_run/lite/Makefile

```diff
@@ -0,0 +1,100 @@
+include ../common.mk
+
+KEY_OPTS       = --keyring-backend=test
+CHAIN_NAME     = local
+CHAIN_OPTS     = --chain-id $(CHAIN_NAME)
+akashctl       = ../../akashctl --home $(CLIENT_HOME) 
+akashd         = ../../akashd --home $(NODE_HOME)
```

Comment:
can use `$(AKASHCTL)`, `$(AKASHD)`

Reply:
fwiw it _is_ nice to have these in `env.sh` so you can source it from the shell for manual interactions.

Reply:
Completely agree, thanks for the reminder! I forgot we had those defined in the common.mk.